tidb-cloud/serverless-high-availability.md

```diff
@@ -0,0 +1,109 @@
+---
+title: High Availability in TiDB Cloud Serverless
+summary: Learn about the high availability architecture of TiDB Cloud Serverless. Discover Zonal and Regional High Availability options, automated backups, failover processes, and how TiDB ensures data durability and business continuity.
+---
+
+# Overview of High Availability in TiDB Cloud Serverless
+
+TiDB Cloud Serverless is designed with robust mechanisms to maintain high availability and data durability by default, preventing single points of failure and ensuring continuous service even in the face of disruptions. As a fully managed service based on the battle-tested TiDB Open Source product, it inherits TiDB's core high availability (HA) features and augments them with additional cloud-native capabilities.
```

Comment:
```suggestion
TiDB Cloud Serverless provides built-in high availability and data durability through its distributed architecture. Based on the open-source TiDB database, it combines TiDB's native high availability features with cloud-native capabilities to prevent single points of failure and maintain service continuity during disruptions.
```